tests/realtikvtest/loaddatatest/load_data_test.go

```diff
@@ -41,13 +41,6 @@ func (s *mockGCSSuite) prepareAndUseDB(db string) {
 	s.tk.MustExec("use " + db)
 }
 
-func adjustOptions(options string, distributed bool) string {
```

Comment:
Maybe move to realtikvtest/importintotest later

Reply:
we can do it later, have to file a CI pr 😅